src/HH4b/postprocessing/PostProcess.py

```diff
@@ -305,19 +306,13 @@ def load_process_run3_samples(args, year, bdt_training_keys, control_plots, plot
             tempw2, _, _ = corrections.ttbar_SF(year, bdt_events, "Xbb", "H2TXbb")
             txbbsf = tempw1 * tempw2
 
-            # total ttbar correction
-            ttbar_weight = ptjjsf * tau32sf * txbbsf
-
-            # get bdt shape correction for each bdt working point
-            bdtsf_bin1, _, _ = corrections.ttbar_bdtshape("cat2", args.bdt_wps[0] + 0.1)
-            bdtsf_bin2, _, _ = corrections.ttbar_bdtshape("cat2", args.bdt_wps[1])
-            bdtsf_bin3, _, _ = corrections.ttbar_bdtshape("cat2", args.bdt_wps[2])
-            print("bin1 2 3", bdtsf_bin1, bdtsf_bin2, bdtsf_bin3)
+            # inclusive bdt shape correction
+            bdtsf, _, _ = corrections.ttbar_bdtshape("cat2", bdt_events, "bdt_score")
 
-            # get bdt shape correction for each value of bdt score
-            bdtsf, bdtsf_up, bdtsf_dn = corrections.ttbar_bdtshape("cat2", bdt_events["bdt_score"])
+            # total ttbar correction
+            ttbar_weight = ptjjsf * tau32sf * txbbsf * bdtsf
```

Comment:
@cmantill should we correct the central value of the `ttbar_weight` with the `bdtsf` or not?

Reply:
@jmduarte let's not apply it for now since the correction is very high in the last bin (almost a factor of 2) because of the low-stats.